src/components/poolstats/PoolPieChartWidget.tsx

```diff
@@ -33,68 +25,20 @@ const PieChartContainer = styled.div`
   transform: rotate(90deg);
 `;
 
-const useMove = () => {
-  const [state, setState] = useState<MouseMoveData>({
-    x: 0,
-    y: 0,
-    r: 0,
-    theta: 0,
-    isActive: false,
-  });
-
-  const handleMouseMove: MouseEventHandler<HTMLDivElement> = (e) => {
-    const currentTarget = e.currentTarget;
-    const boundingRect = currentTarget.getBoundingClientRect();
-
-    // Compute pointer's offset from center of element (in pixels)
-    const offsetX =
-      e.clientX - (boundingRect.x + currentTarget.offsetWidth / 2);
-    const offsetY =
-      e.clientY - (boundingRect.y + currentTarget.offsetHeight / 2);
-
-    // Non-dimensionalize it (-1 to 1)
-    const percentX = (offsetX * 2) / currentTarget.offsetWidth;
-    const percentY = (offsetY * 2) / currentTarget.offsetHeight;
-
-    const r = Math.hypot(percentX, percentY);
-    let theta = (Math.atan2(-percentX, percentY) * 180) / Math.PI;
-    if (theta < 0) theta += 360;
-
-    setState((state) => {
-      const data: MouseMoveData = {
-        ...state,
-        x: percentX,
-        y: percentY,
-        r,
-        theta,
-        isActive: true,
-      };
-
-      return data;
-    });
-  };
-
-  const handleMouseLeave: MouseEventHandler<HTMLDivElement> = (e) => {
-    setState((state) => ({ ...state, isActive: false }));
-  };
-
-  return {
-    mouseData: state,
-    handleMouseMove,
-    handleMouseLeave,
-  };
-};
-
 // MARK: Pie chart setup -------------------------------------------------------------
 
 type PieChartSlice = {
+  index: number;
   percent: number;
   color: string;
+  category: string;
+  metric?: string;
```

Comment:
However, I'm not sure why category and metric should be part of this type. If we were to create another pie chart elsewhere, would these things stick around, or are they specific to this use-case? If specific to this use-case, I think we should `extend` `PieChartSlice` to something like `AllocationPieChartSlice` and put the extra props there